SECURITY.md

```diff
@@ -1,5 +1,178 @@
-# Security Policy
+# Intro
 
-Sui is pre-release software under rapid development. It has not been audited and is not yet ready for production use.
+We appreciate your participation in keeping the Sui network secure. Please read all details on how to report an issue, what bug bounties are available, and who is eligible for bounties.
 
-After we have completed internal and external audits of Sui, we will establish a responsible disclosure policy and a bug bounty program covering both [protocol security](doc/src/learn/sui-security.md) and software security. In the meantime, please report security problems to security@mystenlabs.com.
+The program is run in partnership with [Immunefi](https://immunefi.com/). More information is available on [their site](https://immunefi.com/bounty/sui/).
+
+If you have any questions or concerns, please contact support@immunefi.com.
+
+# Reporting and response process
+
+All reports must be sent using Immunefi’s [secure dashboard](https://bugs.immunefi.com/). **Please DO NOT report a security issue using GitHub, email, or Discord.**
```

Comment:
Suggest removing "please" throughout. It's been shown that users are more likely to disregard as it seems like a suggestion they don't need to follow.

Reply:
thanks.